Library/src/Multiplayer/SpaceEntity.cpp

```
@@ -388,6 +400,10 @@ void SpaceEntity::MarkForUpdate()
         EntitySystem->QueueEntityUpdate(this);
 #endif
     }
+    else
+    {
+        CSP_LOG_MSG(csp::systems::LogLevel::Warning, "Space Entity not marked for update, no local EntitySystem found.");
```

[Comment] Took me a while to track down that in the first iteration, I hadn't injected the EntitySystem, so adding a log for that case as it doesn't seem like it's ever a good thing.

---

I didn't realize that.

The thing that happened here is that when I wrote my new method to construct the SpaceEntity, I just forgot to pass the EntitySystem, so it was silently not queuing the entity update.

My intent here was to make that programming error less easy to make in the future, but you're saying there's a case where _should_ just silently do nothing? _Bit_ of a smell imo, but I'll revert this log if that's the case.

---

I believe its currently used for tested, so we can change a space entity when its not attached to a system